### DISCUSSION TITLE
[✅Completed] 24/7 and autoqueue in config

### DISCUSSION THREAD

**Discussion by DevAmiyo** [SudhanPlayz/Discord-MusicBot#900] [Ideas]
Is it possible to set the default values of `twentyFourSeven` and `autoQueue` in config.js ?

> ✅ Feature added! 6b2552e

**Comment by Neko-Life** [comment]
https://github.com/SudhanPlayz/Discord-MusicBot/blob/master/botconfig.js#L14 for 24/7, for autoQueue not yet possible but it's easy to code it yourself and make a pull request for it if you can

**Reply by DevAmiyo** [reply]
That's for v4.. I meant how to do the 24/7 in config for v5

Edit: I actually did it for v5 before but after 3b5b59f it doesn't work anymore.

**Reply by Neko-Life** [reply]
oh right. Didn't notice i opened the master branch, anyway it's fairly easy to code. And you might wanna do it yourself :>

We encourage users to contribute :heart:

**Reply by DevAmiyo** [reply]
k I'll try

**Reply by Saud-97** [reply]
> Make sure you open up the master branch next time!

what are you talking about?

**Reply by DevAmiyo** [reply]
> Make sure you open up the master branch next time!

uh what?
